### PR TITLE
Maps conflict cholera

### PR DESCRIPTION
**Pull request by zackarno**
Projects still largely in brainstorming phase but with a sense of urgency because they want analysis reviewed by other partners by the 15th. Let's DM off github so I can catch you up on latest conversations.

Given the urgency, I am very open and welcoming of suggestions for additional analysis ideas but i think it might make sense to pose them using the [discussion](https://github.com/OCHA-DAP/ds-som-risk-analysis-support/discussions) feature OR on slack I just initiated for this repo. I think that will allow us to keep the branches tighter in scope and allow us to move more quickly with validating/merging given the timeline. Basically I want the PRs/reviews to prioritize catching any glaring errors in code &  fixing if necessary -- and we save more brainstormy type stuff for discussions?

Let me know if this makes sense and if you have any questions on the approach and/or code

**FYI** 

- don't worry about anything from appendix onwards in the ACLED rmd. It's not really being considered for this project at the moment. I realized I had an exploratory  ACLED R script I wanted to get rid of, but didn't want to get rid of all the code (yet).
- At this stage we are not concerned about polishing any visuals as we will have more time to do that after the 15th and it might even be someone from country-office who does that based on our analyses

### CODE REVIEW

**Comment by zackarno** [comment]
Thanks for all the useful suggestions on slack! I just pushed a commit where I integrate the `ACLED` `actor` attributes so that I can classify conflict events as `Interclan` vs `Non-Intl Armed (Shabaab)`.  I also updated the slides.

My plan is to work w/ the slides and the repo in parallel - right now it's just brainstorming with everything in the `exploration` folder and almost all (if not all) plots in the slides. As I trim down the slides based on internal and external feedback - I will cut the relevant code from the exploration documents into a notebook(s) that will go into an `analysis` folder.

**Comment by caldwellst** [comment]
Nice, for now, won't review anything until ready, and even then, will just be the analysis, don't think there's a need for full detailed code review on the exploration.

**Review by caldwellst — CHANGES_REQUESTED** [review]
Hey hey! Cholera looks good, methods are obviously much simpler and check out. I think a valuable visualisation might be comparing year on the trends by plotting with epiweek on the x axis. Can make for easier yearly comparison that way. But probably not worth putting much more effort into it.

I did request some changes to the conflict stuff. Not sure why `riots` was used as one of the violent events in the filter, but that isn't normally one of them, the violent event missing is for remote violence/explosives (IEDs, drones, missile strikes, that kinda thing), which is definitely going to have at least some cases of in Somalia.

Otherwise, looks good!

File: analysis/01_Conflict_ACLED_slides.Rmd [hunk 1]
```diff
@@ -0,0 +1,614 @@
...
+
+
+ACLED conflict fatalities by type (Shabaab vs Interclan)
+```{r}
+df_adm1_conflict_acled_compare %>%
+  ggplot(
+    aes(x = admin1, y = diff, fill = diff_class)
+  ) +
+  geom_bar(stat = "identity") +
+  scale_y_continuous(breaks = seq(-600, 700, 50)) +
```
[the remark this answers is not in the document]
Comment (caldwellst) [reply analysis/01_Conflict_ACLED_slides.Rmd:158]: They work, obviously, just that they don't reach the max values.

File: analysis/01_Conflict_ACLED_slides.Rmd [hunk 2]
```diff
@@ -0,0 +1,614 @@
...
+  email.address = Sys.getenv("ACLED_USER_EMAIL"),
+  end.date = Sys.Date(),
+  all.variables=T
+)
+
+# filter to just violent events
+df_acled_violent <- df_acled_long %>%
+  filter(event_type %in% c(
+    "Battles",
+    "Riots",
```
[the remark this answers is not in the document]
Comment (zackarno) [reply analysis/01_Conflict_ACLED_slides.Rmd]: woops - nice catch! [60de127](https://github.com/OCHA-DAP/ds-som-risk-analysis-support/pull/2/commits/60de1271a0b75857fbc2452955b621074d79010b)

File: analysis/01_Conflict_ACLED_slides.Rmd [hunk 3]
```diff
@@ -0,0 +1,614 @@
...
+```
+
+# Conflict Bar Charts
+
+## All Fatalities 2022 vs 2023
+
+```{r}
+df_acled_violence_gte2022 <- df_acled_violent %>%
+  filter(year %in% c(2022, 2023),
+         !(year == 2022 & event_date >= "2022-09-29")
```
[the remark this answers is not in the document]
Comment (zackarno) [reply analysis/01_Conflict_ACLED_slides.Rmd]: yeah that's a good point. I've made them 100 % static as you suggested that way the analysis/graphs created here will be reproducible

[60de127](https://github.com/OCHA-DAP/ds-som-risk-analysis-support/pull/2/commits/60de1271a0b75857fbc2452955b621074d79010b)

File: analysis/01_Conflict_ACLED_slides.Rmd (hunk 1 above)
[the remark this answers is not in the document]
Comment (zackarno) [reply analysis/01_Conflict_ACLED_slides.Rmd]: oh yeah I see - good call.   I adjusted them for clearer/improved output

**Review by caldwellst — APPROVED** [review]

**Pull request merged**